tahrir/views.py

```diff
@@ -70,6 +63,15 @@ def _get_user(request, id_or_nickname):
             return None
 
 
+def _get_team(request, team_id):
+    '''Get team for the given team_id'''
+    team = request.db.get_team(team_id=team_id)
+
+    if team:
+        return team
+    return None
```

Comment:
What values could team have? A team object or ?
Just curious in which case ``if team`` will return `False` while the team object isn't a `None` already :)